docs/sessions.md

```diff
@@ -164,6 +164,64 @@ result2 = await Runner.run(
 )
 ```
 
+### SQLAlchemy-powered sessions
```

Comment:
can you move this document change to a separate pull request? with that, we can merge this PR immediately and then we can merge the document changes after the release